Documentation on DEBUG_CONTAINER (#212)
mendix/cf-mendix-buildpack: README.md

You can find the list of available tags here: https://github.com/mendix/cf-mendix-buildpack/tags

Troubleshooting (Rescue mode)
====

> Sometimes the app won't run cause it exists with status code 143. Or for whatever reason the app won't start so that you can debug the issue from within the container. For these cases we have introduced a `DEBUG_CONTAINER` mode. To enable it:

Reviewer: ```suggestion
Sometimes the app won't run because it exits with status code 143. Or, for any reason, the app is unable to start, leaving you unable to debug the issue from within the container. For these cases we have introduced a `DEBUG_CONTAINER` mode. To enable it:
```

```
cf set-env <YOUR_APP> DEBUG_CONTAINER true
cf restart <YOUR_APP>
```

> Now your app starts and you can troubleshoot the problem with:

Reviewer: ```suggestion
Now your app will start in CloudFoundry (n.b. - the Mendix Runtime will not start yet) and you can troubleshoot the problem with:
```

```
cf ssh <YOUR_APP>
export HOME=$HOME/app # this should not be needed but for now it is
export DEBUG_CONTAINER=false # while we are in the container turn it off, we could try to make this optional by detecting other environment variables that are present over ssh but not regular start
export PORT=1234 # so that nginx can start correctly
cd app
python3 start.py
```

> After you are done turn it off with:

Reviewer: ```suggestion
After you are done, you can disable debug mode with:
```
